buildSrc/src/main/kotlin/androidx/build/LibraryVersions.kt

```diff
@@ -90,7 +90,7 @@ object LibraryVersions {
     val MEDIAROUTER = Version("1.3.0-alpha01")
     val NAVIGATION = Version("2.4.0-alpha01")
     val NAVIGATION_COMPOSE = Version("1.0.0-alpha09")
-    val PAGING = Version("3.0.0-beta02")
+    val PAGING = Version("3.1.0-alpha01")
```

Comment:
Not really sure what I was supposed to do here

Reply:
Actually you can just revert this change - I'll handle the version bump in a separate PR and we won't be able to change this until the last beta is scheduled (which also means we'll have to wait to merge this PR).

Reply:
Is it sufficient to increment this locally for the purpose of running `./gradlew updateApi`, and just not commit it? `CONTRIBUTING.md` is a bit unclear.

Reply:
We will unfortunately need to wait for next version bump to happen, and then rebase this PR and run `./gradlew updateApi` as we don't have a separate branch to track 3.1-alpha changes in parallel with 3.0-beta for reasons, but once paging goes rc, that'll happen on a release branch and I will re-ping here.